### PR TITLE
Create livewire-disable-poll-when-hidden.md

### PR DESCRIPTION
**Pull request by abrardev99**

### CODE REVIEW

File: content/tips/livewire-disable-poll-when-hidden.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: 'Livewire Disable Poll When Hidden'
```
Comment (stancl) [inline content/tips/livewire-disable-poll-when-hidden.md:2]:
```suggestion
title: 'Livewire Disable Poll When Hidden'
title: 'Disable Livewire polling when an element is hidden'
```

File: content/tips/livewire-disable-poll-when-hidden.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tweet_id: '1383079668358725636'
+thread_slug: ''
+author_username: abrardev99
+images:
+    - 'https://i.imgur.com/wyVHPhW.png'
```
Comment (stancl) [inline content/tips/livewire-disable-poll-when-hidden.md:7]: Could you remove the margin on this image?

Comment (abrardev99) [reply content/tips/livewire-disable-poll-when-hidden.md]: I updated the image.

**Comment by stancl** [comment]
Thanks for contributing!

**Pull request merged**